### DISCUSSION TITLE
Wrong data type

### DISCUSSION THREAD

**Discussion by dimkene** [prometheus/snmp_exporter#957] [Q&A] [answered]
How can i change the display data type in snmp_exporter?
In the configuration, I specified the data type DisplayString for the sysDescr parameter

```
- name: sysDescr 
    oid: 1.3.6.1.2.1.1.1.0
    type: DisplayString
```

But in the metrics themselves, I still get the gauge data type:
```
# HELP sysDescr A textual description of the entity - 1.3.6.1.2.1.1.1.0
# TYPE sysDescr gauge
sysDescr{sysDescr="RouterOS RB760iGS"} 1
```
Because of this, I get an incorrect display, but I would like something like 
sysDescr "RouterOS RB760iGS"
or
sysDescr RouterOS RB760iGS

**Comment by SuperQ** [comment, marked as answer]
This is expected, Prometheus uses labels to represent strings. Metric values must always be a number.
